src/ui/game_window.h

```diff
@@ -28,14 +29,15 @@
 
 extern int frameCount;
 extern bool isGamePause;
+extern bool isPauseMenuOpen;
 
 void startGame();
 
 void resetGameWindow();
 
 void endReady();
 
-void startReady();
+void startReady(int initialCount);
```

Comment:
i think that we can find a better name for this argument like
`duration`
(example)

Reply:
fe4084c